pages/dropdown-page.ts

```diff
@@ -0,0 +1,51 @@
+import { Page } from "@playwright/test";
+import { BasePage } from "../shared/base-page";
+
+export class DropdownPage extends BasePage {
+  get multiSelectList() {
+    return "#multi-select";
+  }
+
+  get printAllButton() {
+    return 'div.input-body button[id="printAll"]';
```

Comment:
for id there is different way to write it, which you already used

Reply:
Do you mean:
return 'div.input-body #printAll' ?